client/src/routes/automation/esidomBlocks.ts

```diff
@@ -16,18 +16,27 @@ export interface BlocksDefinitions {
     color_picker: any;
     color_rgb: any;
     objet_action: any;
+    binary_sensor: any;
+    person: any;
+    weather: any;
+    zwave: any;
+    sensor: any;
+    light: any;
+    automation: any;
+    switch: any;
+    media_player: any;
     jsonInit:(a: any)=> void;
 }
 
```

Comment:
What does mean this `any` anywhere ?

Reply:
any => because we could not find the matching Blockly type